frontend/src/components/Footer.tsx

```diff
@@ -133,10 +133,10 @@ const Footer: FC<FooterProps> = ({
             <h3 className="sr-only" id="footerNav">
               {footerNavHeader}
             </h3>
-            <div className="flex items-end justify-between md:items-center">
-              <ul className="hidden flex-col flex-wrap space-y-3 marker:text-xs md:list-inside md:list-disc md:flex-row md:items-center md:space-y-0 lg:flex">
+            <div className="flex items-end justify-between sm:items-center">
+              <ul className="flex flex-col gap-2 sm:flex-row marker:text-xs sm:list-inside sm:list-disc">
```

Comment:
Class sorting and remove gap on sm and up breakpoints.
https://tailwindcss.com/blog/automatic-class-sorting-with-prettier
```suggestion
              <ul className="flex flex-col gap-3 marker:text-xs sm:list-inside sm:list-disc sm:flex-row sm:gap-0">
```

Reply:
I removed the mr-4 which negates the need to add sm:gap-0.  Footer link has been removed.  Footer logo styles changed to account for viewport.